pkgs/by-name/zs/zsh-fzf-tab/package.nix

```diff
@@ -93,7 +97,7 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/Aloxaf/fzf-tab";
     description = "Replace zsh's default completion selection menu with fzf!";
     license = lib.licenses.mit;
-    maintainers = with lib.maintainers; [ vonfry ];
```

Comment:
It's not very nice to remove maintainers from the list without their permission! Please add @Vonfry back.

Reply:
We got in touch in a Telegram group, he told me he'll no longer use this package after switching to fish, and asked me to remove him from maintainers.

Reply:
They'll still need to approve this pull request using their GitHub account.